### DISCUSSION TITLE
Add point totals to progression widget

### DISCUSSION THREAD

**Discussion by UnliquidSnake** [RetroAchievements/RAWeb#2028] [Ideas]
In the first column of whatever game is displayed, add a line showing how many points the player has earned and how many total points are in the set, like this:

Game Name
A of B achievements
**X of Y points**
Last played "insert date"

Currently, the only place to see your points in a game is on the game page, which is a lot less convenient than how it was previously, where you could see it in your Last Played entries.

**Comment by wescopeland** [comment]
https://github.com/RetroAchievements/RAWeb/pull/2029

**Reply by wescopeland** [reply]
Reopened, as #2029 only covers the user profile.

**Comment by wescopeland** [comment]
Closing in favor of https://github.com/RetroAchievements/RAWeb/discussions/2048.
